clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiConverter.groovy

```diff
@@ -354,7 +354,11 @@ class KubernetesClientApiConverter {
     }
 
     def kubernetesTcpSocketAction = new KubernetesTcpSocketAction()
-    kubernetesTcpSocketAction.port = tcpSocket.port.toInteger() ?: 0
+    try {
+      kubernetesTcpSocketAction.port = tcpSocket.port.toInteger() ?: 0
```

Comment:
What happens if you try to use the Spinnaker UI to clone a server group that uses a named port?

Reply:
I tried the behavior with the below steps,

1. Deploy ReplicaSet (new server group) with named ports
2. Clone created server group to new one

then confirmed that all named ports were cloned properly with name.

Reply:
Sweet!